src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArr = sourceString.split(';');
```

Comment:
[CODE STYLE] - Variable name `styleArr` does not convey the contents of the variable clearly. Consider using a more descriptive name, such as `styleDeclarations` or `styleRules`.